static/css/style.css

```diff
@@ -586,7 +547,7 @@ select {
 .post__content h1, .post__content h2, .post__content h3, .post__content h4,
 .post__content h5, .post__content h6, .post__content p,
 .post__content blockquote, .post__content .row,
-.post__content .mr-video-container {
```

Comment:
Small mistake? It is necessary to remove `.post__content .mr-video-container` instead `.mr-video-container`